astropy/units/quantity.py

```diff
@@ -1747,6 +1747,17 @@ def isclose(a, b, rtol=1.e-5, atol=None, equal_nan=False, **kwargs):
     Return a boolean array where two arrays are element-wise equal
     within a tolerance.
 
+    .. note::
+
+        `astropy.units.isclose()` behaves slightly different than
```

Comment:
:+1: to the idea, but can we add it to `Notes` below (in both cases)? And maybe make it shorter, e.g., append "It differs from the numpy function in that the default for the absolute tolerance is not ``atol=1e-8``, but zero, as there is no natural way to set a default given two inputs that may have differently scaled units."

Reply:
> can we add it to Notes below (in both cases)?

I wanted to make it more prominent, but fine by me as long as @nstarman is happy with moving this description down.

> maybe make it shorter

Sure